docker/serverless/gcloud_build_image

```diff
@@ -92,12 +92,15 @@ if [ -z "${ESP_FULL_VERSION}" ]; then
 fi
 echo "Building image for ESP version: ${ESP_FULL_VERSION}"
 
-cd "$(mktemp -d /tmp/docker.XXXX)"
+tempdir="$(mktemp -d /tmp/docker.XXXX)"
```

Comment:
No need to change?

Reply:
The fix is to check the status of "mktemp".  Before the fix, it is not checked.